test/sim/abilities/wonderguard.js

```diff
@@ -43,4 +43,12 @@ describe('Wonder Guard', function () {
 		battle.setPlayer('p2', {team: [{species: "Reshiram", ability: 'turboblaze', moves: ['fusionflare']}]});
 		assert.hurts(battle.p1.active[0], () => battle.makeChoices('move sleeptalk', 'move fusionflare'));
 	});
+
+	it('should allow first turn of Sky Drop but not the second', function () {
+		battle = common.createBattle();
+		battle.setPlayer('p1', {team: [{species: "Smeargle", ability: 'wonderguard', moves: ['sleeptalk']}]});
+		battle.setPlayer('p2', {team: [{species: "Aerodactyl", ability: 'owntempo', moves: ['skydrop']}]});
+		battle.makeChoices('move sleeptalk', 'move skydrop');
+		assert.false.hurts(battle.p1.active[0], () => battle.makeChoices('move sleeptalk', 'move skydrop'));
```

Comment:
I prefer not to use `assert.hurts` in general. You can just check if HP equals max HP.

Reply:
Okay, i'll do the changes tomorrow after classes, adding the side effect move (howl) and this change